src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -4942,7 +4942,7 @@ public void foo()
 
         try
         { }
-        catch(System.Exception e)
+        catch(System.Exception e) when(true)
```

Comment:
There is no options on how the space before `when` should behave, currently defaults forcing 1 space if same line, I left it as.